pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -555,6 +558,26 @@ func convertRbacRulesToFilterConfig(service *serviceMetadata, option rbacOption)
 	return &http_config.RBAC{Rules: rbac}
 }
 
+// appendRule appends a |rule| to |rules| if |rule| is not nil.
+func appendRule(rules *policyproto.Permission_AndRules, rule *policyproto.Permission) {
+	if rule == nil {
```

Comment:
[nit] Reverse the condition and put the append inside the if is more readable.

Reply:
Thanks! I will make the change in the next PR.